live-examples/js-examples/object/object-hasown.js

```diff
@@ -0,0 +1,13 @@
+let object1 = {};
+object1.prop = 'exists';
+if (Object.hasOwn(object1, "prop")) {
+  console.log("'prop' is own property") // expected output: 'prop' is own property
+}
+// Property values can be null or undefined
+object1.prop_value_null = null;
+console.log(Object.hasOwn(object1, 'prop_value_null'));  // expected output: true
+object1.prop_value_undefined = undefined;
+console.log(Object.hasOwn(object1, 'prop_value_undefined'));  // expected output: true
+// Inherited and undeclared values return false
+console.log(Object.hasOwn(object1,'toString'));  // expected output: false (inherited)
+console.log(Object.hasOwn(object1,'prop_not_defined'));  // expected output: false (not defined)
```

Comment:
```suggestion
console.log(Object.hasOwn(object1,'prop_not_defined'));  // expected output: false (not defined)

```

Reply:
I think the build is failing because there's no new line at the end here - hoping this suggestion will fix 👍

Reply:
Lets give it a go! Thanks